src/index.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 OpenStax Poland
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license text.
+
+export * from './handlers'
+export * from './interfaces'
+export * from './plugin'
+export * from './util'
+
+export * as CNXML from './cnxml'
+
+export { default as Transforms } from './transforms'
```

Comment:
Those could either extends `Slate~Transforms` or be names `CnxTransforms`

Reply:
I don't like either of those. While extending Slate's `Transforms` would help with name collision when working with just designer, it does nothing when using multiple plugins, since each of them would still have separate `Transforms` (and presumably each of them would extend Slate's). `CnxTransforms` on the other hand is a tautology; `Transforms` from `cnx-designer` obviously contains CNX-related transforms.

In my opinion the correct solution to a name conflict when using this package is to either rename it when importing

```ts
import { Transforms } from 'slate'
import { Transforms as CnxTransforms } from 'cnx-designer'
```

or to use a namespaced import

```ts
import { Transforms } from 'slate'
import * as CNX from 'cnx-designer'

Transforms.slateTransform(...)
CNX.Transforms.cnxTransform(...)
```